main.js

```diff
@@ -159,4 +163,25 @@ btns.forEach((btn) => {
 
 closeBtn.addEventListener('click', () => {
   popup.classList.add('hide');
-});
\ No newline at end of file
+});
+
+// Implement the client validation for inputs
```

Comment:
- [ ] Please check the validations, because If a user submits the contact form with the email field containing only lowercase characters the form should be sent normally, however in your case error keeps posing up:

![chrome-capture (1)](https://user-images.githubusercontent.com/77038610/146432133-04b65189-5b17-423b-aeb7-1ed432409015.gif)

Reply:
Thanks